From: Karol Mroz <kmroz@suse.com>
Date: Mon, 2 May 2016 14:01:27 +0200
Subject: global: don't link lttng into libglobal

Rely on dynamic initialization instead. Linking lttng in this way had
the unfortunate side effect of causing radosgw to segfault (when
daemonized) during sigterm processing (ie. during lttng_ust_exit()).

This was originally removed in 638738f, but accidentally re-added via
5f61d36.

Signed-off-by: Karol Mroz <kmroz@suse.com>
(cherry picked from commit 37ecfa6c754683cac1c6f4c66a1a491d2fd5db1d)

Resolves: rhbz#1336877
---
 src/global/Makefile.am | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/global/Makefile.am b/src/global/Makefile.am
index ed93426..243a7fa 100644
--- a/src/global/Makefile.am
+++ b/src/global/Makefile.am
@@ -6,9 +6,7 @@ libglobal_la_SOURCES = \
 	common/TrackedOp.cc
 
 libglobal_la_LIBADD = $(LIBCOMMON)
-if WITH_LTTNG
-libglobal_la_LIBADD += -ldl -llttng-ust
-endif
+
 noinst_LTLIBRARIES += libglobal.la
 
 noinst_HEADERS += \
